classes/Http/Form/SignupForm.php

```diff
@@ -71,8 +64,7 @@ public function validateAll($action = 'create')
             $validSpeakerBio = $this->validateSpeakerBio();
         }
 
-        return
-            $validEmail && $validPasswords && $validFirstName && $validLastName && $validCompany && $validTwitter && $validUrl && $validSpeakerInfo && $validSpeakerBio && $validSpeakerPhoto && $agreeCoc;
+        return $validEmail && $validPasswords && $validFirstName && $validLastName && $validUrl && $validSpeakerInfo && $validSpeakerBio && $validSpeakerPhoto && $agreeCoc;
```

Comment:
Here we use variables and return whether all those are true, but in the ```TalkFrom``` we don't use temporary variables. 
It might be nicer to make both work in the same way, what do you think?

Reply:
In which direction

* [ ]  inline variables
* [ ] introduce variables

?

Reply:
I don't have a preference for one or the other really, which do you think looks better?